tests/unit/amo/reducers/test_versions.js

```diff
@@ -61,6 +65,22 @@ describe(__filename, () => {
     ]);
   });
 
+  describe(createInternalVersion, () => {
+    it('returns an object with the expected AddonVersionType', () => {
+      expect(createInternalVersion(fakeVersion)).toEqual({
+        compatibility: fakeVersion.compatibility,
+        platformFiles: createPlatformFiles(fakeVersion),
+        id: fakeVersion.id,
+        license: {
+          name: fakeVersion.license.name,
+          url: fakeVersion.license.url,
+        },
+        releaseNotes: fakeVersion.release_notes,
+        version: fakeVersion.version,
+      });
+    });
+  });
+
   describe(getLoadingBySlug, () => {
```

Comment:
Hmm. I guess this file wasn't using string specs? I think it should be a string.